src/main/java/am/hsp/apps/paint/drawing_tools/DrawingTool.java

```
@@ -0,0 +1,98 @@
+package main.java.am.hsp.apps.paint.drawing_tools;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import main.java.am.hsp.apps.paint.exceptions.InvalidBrushException;
+
+/**
+ * Conatins attributes related to the drawing tools.
+ * 
+ * @author user
+ *
+ */
+public class DrawingTool {
+
+    private static final int NUMBER_OF_BRUSHES = 3;
+    private static final int THIN_BRUSH_SIZE = 1;
+    private static final int AVERAGE_BRUSH_SIZE = 2;
+    private static final int THICK_BRUSH_SIZE = 3;
+    private static final int THIN_BRUSH_ID = 1;
+    private static final int AVERAGE_BRUSH_ID = 2;
+    private static final int THICK_BRUSH_ID = 3;
+
+    private static final int[] BRUSHES_SIZE = { THIN_BRUSH_SIZE, AVERAGE_BRUSH_SIZE, THICK_BRUSH_SIZE };
+    private static final int[] BRUSHES_ID = { THIN_BRUSH_ID, AVERAGE_BRUSH_ID, THICK_BRUSH_ID };
+
+    private List<Brush> brushes;
+
+    /**
+     * Constructor for initializing the drawing tools.
+     */
+    public DrawingTool() {
+        this.brushes = new ArrayList<>();
+
+        for (int i = 0; i < NUMBER_OF_BRUSHES; i++) {
+            this.brushes.add(new Brush(BRUSHES_SIZE[i], BRUSHES_ID[i]));
+        }
+    }
+
+    /**
+     * Returns the brush with the given id, if it exists, otherwise throws
+     * exception.
+     * 
+     * @param brushId
+     * @return
+     */
+    public Brush getBrushById(int brushId) {
+        for (Brush brush : brushes) {
+            if (brush.getId() == brushId) {
+                return brush;
+            }
+        }
+
+        throw new InvalidBrushException();
+    }
+
+    /**
+     * Checks whether there is a brush of the given id in the drawing tool. If
+     * no, throws exception.
+     * 
+     * @param brushId
+     */
+    public void validateBrush(int brushId) {
```

No, no exception handling should be done here. Method just need to throw an exception when invalid brush id is passed...
Error handling should be done in the class which is reponsible for communicating with the user...